backend/de.metas.cucumber/src/test/java/de/metas/cucumber/stepdefs/invoicecandidate/C_Invoice_Candidate_StepDef.java

```diff
@@ -152,8 +156,11 @@ public void recompute_invoice_candidates_if_required(@NonNull final DataTable da
 
 			if (recompute)
 			{
-				final InvoiceCandidateIdsSelection onlyInvoiceCandidateIds = InvoiceCandidateIdsSelection.ofIdsSet(
-						ImmutableSet.of(InvoiceCandidateId.ofRepoId(invoiceCandidate.getC_Invoice_Candidate_ID())));
+				final ImmutableSet<InvoiceCandidateId> invoiceCandidateIds = ImmutableSet.of(InvoiceCandidateId.ofRepoId(invoiceCandidate.getC_Invoice_Candidate_ID()));
+
+				invoiceCandDAO.invalidateCandsFor(invoiceCandidateIds);
+
```

Comment:
dev-note: normally, we don't need this for our cases due to `de/metas/invoicecandidate/async/spi/impl/InvoiceCandWorkpackageProcessor.java:121` i.e.:

```JAVA
invoiceCandDAO.invalidateCands(candidatesOfPackage);
```

but I guess there might be cases in the future where we would use this step def to recompute candidates at a different timing so I thought it makes sense